src/libraries/System.Private.CoreLib/src/System/Threading/Lock.cs

```diff
@@ -1265,11 +1490,37 @@ public static bool TrySetIsWaiterSignaledToWake(Lock lockObj, State state)
             }
         }
 
+
+        internal struct ThreadId
```

Comment:
Is this wrapper still needed now that thread id is managed thread id? Also, can we make it `int` since that's what the managed thread ID is in the public surface?

Reply:
We can remove this wrapper.